src/complexity_measures/missing_dispersion.jl

```diff
@@ -0,0 +1,65 @@
+using DelayEmbeddings
+export MissingDispersionPatterns
+
+"""
+    MissingDispersionPatterns <: ComplexityMeasure
+    MissingDispersionPatterns(; est = Dispersion())
```

Comment:
```suggestion
    MissingDispersionPatterns(est = Dispersion())
```
I would argue that since this takes a single argument, making it an optional positional argument rather than a keyword argument makes more sense.

Reply:
Agreed.